makeown.lua

```diff
@@ -215,6 +262,8 @@ function make_own(unit)
     else
         unit.flags1.tame = true
         unit.training_level = df.animal_training_level.Domesticated
+        -- No more "vengeful" spam when civilizing Giants!
+        unit.enemy.caste_flags.LARGE_PREDATOR = false;
```

Comment:
does this "stick"? I thought `unit.enemy.caste_flags` gets overwritten with actual race caste flags (plus syndrome flags) periodically

Reply:
yah it doesn't stick, might as well remove it